tests/integration/targets/inventory_aws_rds/aliases

```diff
@@ -1,2 +1,2 @@
+time=3m
```

Comment:
3m is a bit optimistic IMO.

Reply:
The integration has been rewritten to create only one RDS instance to work with, I think 3m will be enough but let see